prisma/schema.prisma

```diff
@@ -76,6 +76,7 @@ model Transaction {
   updatedAt                DateTime          @updatedAt
   txHash                   String
   status                   TransactionStatus
+  confirmations            Int               @default(0)
```

Comment:
lets have it nullable, so the default case and for rolled back and similar is null (as 0 confirmations is a valid state for a confirmed state on Cardano)